xrdp/xrdp_mm.c

```diff
@@ -1042,27 +1042,110 @@ dynamic_monitor_data_first(intptr_t id, int chan_id, char *data, int bytes,
     return 0;
 }
 
+/******************************************************************************/
+static int
+process_dynamic_monitor_description(struct xrdp_wm *wm, struct display_size_description *description) {
+    int error = 0;
+    struct xrdp_mm* mm = wm->mm;
+    struct xrdp_mod* module = mm->mod;
+
+    if (wm->client_info->suppress_output == 1) {
+        LOG(LOG_LEVEL_INFO, "process_dynamic_monitor_description: Not allowing resize. Suppress output is active.");
+        return 0;
+    }
+    if (description->session_width <= 0 || description->session_height <= 0) {
+        LOG(LOG_LEVEL_INFO, "process_dynamic_monitor_description: Not allowing resize due to invalid dimensions (w: %d x h: %d)", description->session_width, description->session_height);
+        return 0;
+    }
+
+    // TODO: Unify this logic with server_reset
+    error = libxrdp_reset(wm->session, description->session_width, description->session_height, wm->screen->bpp);
+    if (error != 0)
+    {
+        LOG(LOG_LEVEL_INFO, "dynamic_monitor_data: libxrdp_reset failed %d", error);
```

Comment:
this seem like it warrants an error message (as well as all of the other error cases in this function). The rest of the code use the error log level for these situations.

Reply:
I think I fixed this?